src/plugins/sorting-task/SortingTaskNode/ReadView/style.scss

```diff
@@ -0,0 +1,79 @@
+.learning-items{
+  padding: 40px; /* padding to prevent overflow during animation*/
+
+  --border-width: 2px;
+  --border-radius: 6px;
+}
+
+/* inspiration for card-flip animation: https://www.w3schools.com/howto/howto_css_flip_card.asp */
+/* The flip card container - set the width and height to whatever you want. We have added the border property to demonstrate that the flip itself goes out of the box on hover (remove perspective if you don't want the 3D effect */
+.flip-card {
+  background-color: transparent;
+  width: 100%;
+  max-width: 300px;
+  height: 50vh;
+  max-height: 400px;
+  margin: 0 auto;
+  perspective: 1000px;
+}
+
+/* This container is needed to position the front and back side */
+.flip-card-inner {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  text-align: center;
+  transition: transform 0.8s;
+  transform-style: preserve-3d;
+  border: var(--border-width) solid #333;
+  border-radius: var(--border-radius);
+}
+
+/* Do an horizontal flip when you move the mouse over the flip box container */
+/* TODO replace :hover with .flipped or .active class and toggle the class with js on click */
+.flip-card:hover .flip-card-inner {
+  transform: rotateY(180deg);
+}
+
+/* Position the front and back side */
+.flip-card-content {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  margin: 0 !important; /* "!important" to reset all margins applied by default to dt & dd */
```

Comment:
In gneeral, it's considered bad practice to use `!important`. If you really need to override another value, it’s usually better to increase selector specificity only slightly instead of using `!important!`. However, in this case, I don't think you actually need to reset margins this way.

Reply:
Here’s an article by Harry Roberts that explains the rare situations where you might really want to use `!important` and what to do in other situations: https://csswizardry.com/2016/05/the-importance-of-important/